src/providers/completion/styleCompletionItemProvider.ts

```diff
@@ -1,18 +1,15 @@
-import { completion } from 'titanium-editor-commons';
-import project from '../../project';
 import * as related from '../../related';
 import * as utils from '../../utils';
 import * as alloyAutoCompleteRules from './alloyAutoCompleteRules';
 
 import { CompletionItem, CompletionItemKind, CompletionItemProvider, Position, Range, SnippetString, TextDocument, workspace } from 'vscode';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/providers/com...
  
  
  DangerID: danger-id-Danger;
  File: src/providers/completion/styleCompletionItemProvider.ts;
  Line: 5;
-->

- :warning: src/providers/completion/styleCompletionItemProvider.ts line 5 – 'CompletionItemProvider' is defined but never used. (@typescript-eslint/no-unused-vars)

Reply:
Should this be removed too? @ewanharris

Reply:
Yah it should, will push a commit now